java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java

```diff
@@ -18,7 +18,7 @@ public class WebFluxClient implements HttpClient {
   private final WebClient webClient;
 
   public WebFluxClient(String baseUrl) {
-    webClient = WebClient.create(baseUrl);
+    webClient = WebClient.create(baseUrl.endsWith("/") ? baseUrl : baseUrl + "/");
```

Comment:
Why was this change needed?

Reply:
Wouldn't it be more common to prefix the uri paths with a slash instead? I think the underlying UriBuilder would handle any excessive ones, perhaps?

Reply:
Nope, it doesn't. And that's the reason. :-)

Reply:
Scratch that, you're right. Fixed.